htmd/queues/localqueue.py

```diff
@@ -266,10 +292,16 @@ def _getdevices(self):
             devices = range(ngpu)
 
         if devices is None:
-            raise NameError("Could not determine which GPUs to use. "
-                            "Specify the GPUs with the `ngpu=` or `devices=` parameters")
+            raise NameError('Could not determine which GPUs to use. Specify the GPUs with the `ngpu` or `devices` '
```

Comment:
Can you change this to `RuntimeError`. I used to put everything to `NameError` in the past because I thought it was the default error :P